tests/yam/agama/boot_agama.pm

```diff
@@ -52,12 +52,19 @@ sub run {
     my $grub_menu = $testapi::distri->get_grub_menu_agama();
     my $grub_entry_edition = $testapi::distri->get_grub_entry_edition();
     my $agama_up_an_running = $testapi::distri->get_agama_up_an_running();
+    my $profile_url;
 
     # prepare kernel parameters
     if (my $agama_auto = get_var('AGAMA_AUTO')) {
-        my $path = expand_agama_profile($agama_auto);
-        set_var('AGAMA_AUTO', $path);
-        set_var('EXTRABOOTPARAMS', get_var('EXTRABOOTPARAMS', '') . " agama.auto=\"$path\"");
+        # Currently jsonnet is only available to install on x86_64, remove this after include in the rest of architectures.
+        if (is_x86_64() || is_aarch64()) {
+            $profile_url = generate_json_profile();
```

Comment:
here in our case nothing will be expanded, but we still need to provide expansion for other user that directly want to user the profile without composing it in runtime, otherwise would break for them here.